doc/interop-test-descriptions.md

```diff
@@ -944,6 +944,11 @@ the experimental flag, `soak_iterations`.
 This tests puts stress on several gRPC components; the resolver, the load 
 balancer, and the RPC hotpath.
 
+#### long_connection
```

Comment:
One thought BTW: though we chatted earlier and decided on using unary RPC's, on second thought I'm thinking it will also be useful to run a "long connection" bidi streaming test case <i>in addition</i>.... just to test some things that this test won't be as well suited for.

So while this test would be good to have, I think it would <i>also</i> be good to have a single long "bidi streaming" version of it.... which could go into a separate PR, but I'd be open to add it here